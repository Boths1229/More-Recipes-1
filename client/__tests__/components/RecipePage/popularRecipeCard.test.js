import React from 'react';
import { shallow } from 'enzyme';
import PopularRecipeCard from '../../../components/RecipePage/PopularRecipeCard';

let props;

const setup = () => {
  props = {
    recipeName: 'fried rice',
    id: 5,
    image: 'image'
  };

  return shallow(<PopularRecipeCard {...props} />);
};

describe('Popular Recipe card component', () => {
  it('should render correctly', () => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('div').at(2).text()).toBe('fried rice');
  });
});
