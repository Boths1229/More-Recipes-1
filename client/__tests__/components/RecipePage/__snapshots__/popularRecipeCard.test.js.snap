// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popular Recipe card component should render correctly 1`] = `
<div
  className="side_recipes"
>
  <Link
    className="side_item"
    onlyActiveOnIndex={false}
    style={Object {}}
    to="/recipes/5"
  >
    <div
      className="img"
    >
      <img
        alt=""
        src="image"
        style={
          Object {
            "height": "auto",
            "marginLeft": "0px",
            "marginTop": "-13.3571px",
            "width": "100px",
          }
        }
      />
    </div>
    <div
      className="title dot"
      style={
        Object {
          "wordWrap": "break-word",
        }
      }
    >
      fried rice
    </div>
  </Link>
</div>
`;
